packages/core/package.json

```diff
@@ -18,8 +18,7 @@
     "*.js"
   ],
   "scripts": {
-    "test": "nyc --reporter=lcov -- jasmine '!(node_modules)/**/*.test.js'",
-    "test:types": "tsc -p types/test/tsconfig.json && jasmine types/test/test.js"
+    "test:types": "tsc -p types/test/tsconfig.json && jest types/test/test.js"
```

Comment:
Starting jest for a single test file like this is not ideal due to boot up time. I've opted to keep the approach as similar to the existing as possible however, so `npm run test:types` still remains a separate function. This is in the knowledge that it will only be a temporary thing until we add TypeScript.